packages/@react-aria/gridlist/src/useGridListItem.ts

```diff
@@ -291,7 +291,10 @@ export function useGridListItem<T>(props: AriaGridListItemOptions, state: ListSt
   });
 
   if (isVirtualized) {
-    rowProps['aria-rowindex'] = node.index + 1;
+    let {collection} = state;
+    let nodes = [...collection];
+    // TODO: refactor ListCollection to store an absolute index of a node's position?
+    rowProps['aria-rowindex'] = nodes.find(node => node.type === 'section') ? [...collection.getKeys()].filter((key) => collection.getItem(key)?.type !== 'section').findIndex((key) => key === node.key) + 1 : node.index + 1;
```

Comment:
i did end up changing how we calculate rowIndex for rows inside Sections since Safari VO would read out the rows incorrectly otherwise. for example, say you were on the first item in Section 3, instead of saying something like "row 8 of 12" it would say "row 1 of 12"

Reply:
wonder if it would reasonable to use `aria-posinset` like tree does, did you try that before?

Reply:
the way we calculate posinset for tree is also `node.index + 1` so i'm not sure if that would be any different

Reply:
so if we just had `aria-posinset` (so no aria-rowindex) then it seems to announce fine, but if you had a combo of aria-posinset and aria-rowindex where both are calculated node.index + 1, then the announcements are wrong. seems like we should probably have aria-rowindex based on https://w3c.github.io/aria/#aria-rowindex so i think regardless, we'll need to update how rowindex is calculated